src/assets/main.css

```diff
@@ -149,6 +150,10 @@ ul.btn-group {
 .control.radio-group span label {
   font-weight: unset;
 }
+.control.select-group {
```

Comment:
"Select" is a particular HTML element. I don't see this being used with that tag. Try to find a synonym for this group type if possible.

Reply:
OK, will do.

Reply:
for now I replaced "select-group" with "dropdown-group" and also added "checkbox-group" to match the already existing "radio-group". Not sure but it seemed to make sense that a row of each of these HTML elements might need different spacing. Will move this to the new issue https://github.com/xenharmonic-devs/scale-workshop/issues/644.